tests/test_api.py

```diff
@@ -63,10 +62,9 @@ def test_auth_badotp(self):
             self.api.authenticate()
 
     def test_api_auth(self):
-        self.assertTrue(isinstance(self.api.token, collections.Mapping))
-        self.assertTrue("expiration" in self.api.token)
-        self.assertTrue("journalist_uuid" in self.api.token)
-        self.assertTrue("token" in self.api.token)
+        self.assertTrue(isinstance(self.api.token, str))
+        self.assertTrue(isinstance(self.api.token_expiration, datetime.datetime))
+        self.assertTrue(isinstance(self.api.token_journalist_uuid, str))
```

Comment:
Add the same tests in `APIProxy` class too please.

Reply:
I'm going to add https://github.com/freedomofpress/securedrop-sdk/issues/67 to the next sprint for maintainability of this package, let me know if you disagree @kushaldas